types/wordpress__block-serialization-spec-parser/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for @wordpress/block-serialization-spec-parser 3.1
+// Project: https://github.com/WordPress/gutenberg/tree/master/packages/block-serialization-spec-parser/README.md
+// Definitions by: Derek Sifford <https://github.com/dsifford>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+export interface Block {
+    blockName: string | null;
+    attrs: Record<string, any>;
+    innerBlocks: readonly Block[];
```

Comment:
Is there a reason `innerBlocks` is `readonly` while `innerContent` isn't?

Reply:
I think you may have looked at an earlier commit. I think I might have noticed that at the same time you did and I already committed the update.